miner/src/pool/queue.rs

```diff
@@ -138,6 +138,50 @@ impl CachedPending {
 	}
 }
 
+#[derive(Debug)]
+struct RecentlyRejected {
+	inner: RwLock<HashMap<H256, transaction::Error>>,
```

Comment:
Curious about why you're storing the error here as opposed to using a `HashSet`. Do you plan to use the error count stats to refine the logic further down?

Reply:
Thank you for the explanation! :)